src/discljord/events/state.clj

```diff
@@ -136,8 +136,8 @@
   [_ user state]
   (swap! state update ::bot merge user))
 
-(def ^:private caching-handlers
-  "Handler map for all state-caching events."
+(def caching-handlers
```

Comment:
Why not private? It's only used within the `discljord.events.state` namespace.

Reply:
This change is part of a much older commit that hasn't been merged yet, forgot to revert it for this PR. (The idea of this was to allow for custom caching handlers that you could pass to the functions in that namespace, that will still be realised at some point)